src/ansys/rocky/core/launcher.py

```diff
@@ -21,22 +21,23 @@
 # SOFTWARE.
 """Module that exposes functions to launch a Rocky application session."""
 import contextlib
-import os
 from pathlib import Path
 import subprocess
 import time
-from typing import Optional, Union
 
 from Pyro5.errors import CommunicationError
+from ansys.tools.path import get_available_ansys_installations
 
 from ansys.rocky.core.client import DEFAULT_SERVER_PORT, RockyClient, connect_to_rocky
 from ansys.rocky.core.exceptions import RockyLaunchError
 
 _CONNECT_TO_SERVER_TIMEOUT = 60
+MINIMUM_ANSYS_VERSION_SUPPORTED = 242
 
 
 def launch_rocky(
-    rocky_exe: Optional[Union[Path, str]] = None,
+    rocky_exe: Path | str = None,
```

Comment:
I guess the `|` does not work under Python 3.10. If it does, you need to specify the `None` type: `rocky_exe: Path | str | None = None`